src/Stratis.Bitcoin.Features.MemoryPool/Fee/EstimationResult.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Stratis.Bitcoin.Features.MemoryPool.Fee
+{
+    /// <summary>
+    /// Used to return detailed information about a fee estimate calculation
+    /// </summary>
+    public class EstimationResult
+    {
+        public EstimatorBucket Pass { get; set; }
```

Comment:
Using the summary on the class, I have no idea what these four properties are for. Could you add comments explaining their role?